eng/pipelines/runtime.yml

```diff
@@ -233,130 +233,148 @@ jobs:
             eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true),
             eq(variables['isFullMatrix'], true)))
 
-#
-# Build Mono debug
-# Only when libraries or mono changed
+# Build the whole product using Mono runtime
+# Only when libraries, mono or installer are changed
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/mono/templates/build-job.yml
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: ${{ variables.debugOnPrReleaseOnRolling }}
     runtimeFlavor: mono
-    buildConfig: debug
     platforms:
-    - Android_x64
     - Android_x86
-    - Android_arm
     - Android_arm64
     - tvOS_x64
-    - tvOS_arm64
-    - iOS_x64
-    - iOS_x86
-    - iOS_arm
     - iOS_arm64
-    - OSX_x64
-    - Linux_x64
-    - Linux_arm
-    - Linux_arm64
-    - Linux_musl_x64
+    - iOS_x86
     - Browser_wasm
-    # - Linux_musl_arm64
-    - Windows_NT_x64
-    # - Windows_NT_x86
-    # - Windows_NT_arm
-    # - Windows_NT_arm64
+    - Linux_arm
     jobParameters:
+      testGroup: innerloop
+      nameSuffix: All_Mono
+      buildArgs: -s mono+libs+installer -c $(_BuildConfig)
+                 /p:StripSymbols=true /p:SkipTests=true
```

Comment:
Unrelated but I'm wondering why we haven't yet excluded installer.tests from the installer subset? This should save us only ~1min but presumably test dependencies wouldn't need to be restored during the root build.

Reply:
Yeah, let's open an issue for that and we can update the builds to do that whenever we don't want to run/build tests.

Reply:
Actually, when runtimeFlavor: mono, we don't build anything but the pkgproj so I can remove the `SkipTests` property:
https://github.com/dotnet/runtime/blob/master/eng/Subsets.props#L65

Reply:
Opened https://github.com/dotnet/runtime/issues/37436 to track this.